transport/http/transport.go

```diff
@@ -50,6 +54,16 @@ type TransportOption func(*transportConfig)
 
 func (TransportOption) httpOption() {}
 
+// NilOption is empty option - example use case:
```

Comment:
I’ll buy the nil option with this rationale. It’s simpler for the user than resorting to a slice. However, it does establish a precedent of having to create a nil option for every option type.